packages/aws-cdk/test/integ/cli/test-cdk-multiple-toolkit-stacks.sh

```diff
@@ -10,13 +10,22 @@ toolkit_stack_name_1="toolkit-stack-1-${RANDOM}"
 toolkit_stack_name_2="toolkit-stack-2-${RANDOM}"
 
 # deploy two toolkit stacks into the same environment (see #1416)
-cdk bootstrap --toolkit-stack-name ${toolkit_stack_name_1}
-cdk bootstrap --toolkit-stack-name ${toolkit_stack_name_2}
+cdk bootstrap --toolkit-stack-name ${toolkit_stack_name_1} --tags Foo=Bar
+cdk bootstrap --toolkit-stack-name ${toolkit_stack_name_2} --tags Foo=Bar
 
 # just check that the new stack exists
 aws cloudformation describe-stack-resources --stack-name ${toolkit_stack_name_1}
 aws cloudformation describe-stack-resources --stack-name ${toolkit_stack_name_2}
 
+# get tags from the new stack
+tag_stack_1=$(aws cloudformation describe-stacks --stack-name ${toolkit_stack_name_1} --query "Stacks[0].Tags[?Key=='Foo'].Value" --output text)
+tag_stack_2=$(aws cloudformation describe-stacks --stack-name ${toolkit_stack_name_2} --query "Stacks[0].Tags[?Key=='Foo'].Value" --output text)
+
+# check if tag is not equal to bar
+if [ "${tag_stack_1}" != "Bar" ] && [ "${tag_stack_2}" != "Bar" ]; then
```

Comment:
did you mean `||` here? I think either one indicates failure of the test

Reply:
Makes sense. I changed. Thanks